File: ts/report.ts
import fs from 'fs'

const data = fs.readFileSync("student.json" , "utf-8");
const student = JSON.parse(data);

console.log("------STUDENT REPORT----------");
console.log(`Name ${student.student_name}`);

console.log();

console.log("Scores: ")

const subjects = ["Math", "English", "Science", "ICT", "Social Studies"];

subjects.forEach((subject , index ) => {
    console.log(`${subject} : ${student.student_scores[index]}`)
})

console.log();


console.log(`Total: ${student.total_mark}`);
console.log(`Average: ${student.average_mark.toFixed(2)}`);
console.log(`Grade: ${student.student_grade}`);
console.log(`Status: ${student.student_status}`);

File: report.py
import json

with open("student.json" , "r") as file:
    student = json.load(file)

print("-----STUDENT REPORT--------")
print(f" Name {student['student_name']}")
print()

print("Scores:")
subjects = ["Math", "English", "Science", "ICT", "Social Studies"]

for subject, score in zip(subjects, student["student_scores"]):
    print(f"{subject}: {score}")


print()
print(f"Total: {student['total_mark']}")
print(f"Average: {student['average_mark']:.2f}")
print(f"Grade: {student['student_grade']}")
print(f"Status: {student['student_status']}")